bin/key_server.js

```diff
@@ -174,6 +175,11 @@ function main() {
       })
     }
   })
+  .catch((err) => {
+    log.error({ op: 'server.start.1', msg: 'startup caught error',
```

Comment:
Or, just exit in run on bind failure and don't reject? You're dead at that point.